docs/exercises/exercise-list-ch-4.md

```diff
@@ -0,0 +1,74 @@
+# Exercise list _Chapter 4_
```

Comment:
Needs to be linked to the lesson page, and there are exercises on that page that you could add to this one

Reply:
Coolio will do it asap, probably later on when I get home